sematic/ui/packages/main/src/hooks/useMatplotLib.ts

```diff
@@ -1,3 +1,5 @@
+///<reference path="../vendor/types/mpld3.d.ts" />
```

Comment:
What's this?

Reply:
This is to explicit reference the [ambient modules](https://www.typescriptlang.org/docs/handbook/modules.html#ambient-modules) for libraries that do not implement typescript typings properly.